mule-apikit-module/src/test/java/org/mule/module/apikit/validation/attributes/QueryStringTestCase.java

```diff
@@ -84,4 +82,12 @@ public void invalidObjectProperty() {
         .response()
         .when().get("/api/type-property-type?property={\"firstname\": \"Lionel\", \"nickname\": \"Messi\"}");
   }
+
+  @Test
+  public void defaultValueProperty() {
+    given().queryParam("property", "someValue")
+        .expect().statusCode(200)
+        .response()
+        .when().get("/api/default-value-property");
```

Comment:
You are checking request passed ok, but not that default values were added

Reply:
Values are required, so they are being validated